openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -121,6 +122,19 @@ def _create_from_course(cls, course):
             end = ccx.due
             max_student_enrollments_allowed = ccx.max_student_enrollments_allowed
 
+        # Try to create a thumbnail of the course image. If this fails for any
+        # reason (weird format, non-standard URL, etc.), just fall back to the
+        # old behavior of using the originally provided course_image_url().
+        try:
+            course_image_url_ = create_course_image_thumbnail(course, THUMBNAIL_DIMENSIONS)
```

Comment:
Can we keep the thumbnail in a separate field?  The reason is that my [other PR](https://github.com/edx/edx-platform/pull/10841) expands CourseOverview to support both the Course Catalog and Course About API.  For the Course Catalog, the thumbnail image may suffice.  But for Course About, we would want the larger image.